Documentation/Introduction/Index.rst

```diff
@@ -54,15 +54,15 @@ to existing tables and add new tables. Several required extensions that are
 always loaded, already deliver some TCA files in their
 :file:`Configuration/TCA` directories.
 
-Most importantly, the extension "core" comes with a definition of pages,
+Most importantly, the extension "cms-core" comes with a definition of pages,
 be_users and further tables needed by the whole system.
-The extension "frontend" comes with the tables tt_content, fe_users, sys_template and more.
-See the directories :file:`typo3/sysext/core/Configuration/TCA/` and
-:file:`typo3/sysext/frontend/Configuration/TCA/` for the complete list of the TYPO3 CMS tables.
+The extension "cms-frontend" comes with the tables tt_content, fe_users, sys_template and more.
+See the directories :file:`vendor/typo3/cms-core/Configuration/TCA/` and
+:file:`vendor/typo3/cms-frontend/Configuration/TCA/` for the complete list of the TYPO3 CMS tables.
```

Comment:
This is however only true for composer-based installations

Reply:
Ok but still most people hopefully use the Composer based setup. 
Anyhow if I have seen correctly mostly everything I said was not right. You can therefore skip the pull request. Or we write explicitly that this path only is valid in Composer based installations. I don't know more what to add to your comment.

Reply:
> hopefully use the Composer based setup.

@Alagts Sadly, no. Taken the latest steps it's ~47% between composer and non-composer installations for v11 and v12

See: https://www.t3versions.com/statistics-detail/21